packages/fxa-settings/src/components/images/en.ftl

```diff
@@ -0,0 +1,12 @@
+## Images - these are all aria labels used for illustrations
+
+hearts-broken-image-aria-label =
+  .aria-label = A computer and a mobile phone and an image of a broken heart on each
+hearts-verified-image-aria-label =
+  .aria-label = A computer and a mobile phone and a tablet with a pulsing heart on each
```

Comment:
Without looking at the images below (or knowing if these are existing strings that were copy-pasted), I wonder how useful the actual aria labels are to a screen reader. Or if we should maybe swing in the other direction and just say something about "Pairing unsuccessful" or "Successfully paired desktop and mobile device".

But as I am no a11y expert, i'll defer to the skilled observations of @sardesam / @xlisachan / @LZoog and then sit back and learn something.

Reply:
This is a great question! The guiding philosophy for the illustration aria labels in fxa-settings has generally been to try to enrich the experience for those using screen readers in much the way the visual image does. The other option in a11y best practices is to deem the image unnecessary to the experience, and to omit it completely from the screen reader experience by applying `aria-hidden={true}`, as seen in the `Image` component defined in this PR. We wouldn't want to have an `aria-label` that says "Pairing successful" etc because it repeats information that's already present on the page and would just be a repeat for the user.